source/includes/common-installation.rst

```diff
@@ -189,11 +189,18 @@ For example:
 
    $ nano /etc/fstab
 
-     # <file system>  <mount point>  <type>  <options>                <dump>  <pass>
-     LABEL=DISK1      /mnt/disk1     xfs     defaults,noatime,nofail  0       2
-     LABEL=DISK2      /mnt/disk2     xfs     defaults,noatime,nofail  0       2
-     LABEL=DISK3      /mnt/disk3     xfs     defaults,noatime,nofail  0       2
-     LABEL=DISK4      /mnt/disk4     xfs     defaults,noatime,nofail  0       2
+     # <file system>  <mount point>  <type>  <options>         <dump>  <pass>
+     LABEL=DISK1      /mnt/disk1     xfs     defaults,noatime  0       2
+     LABEL=DISK2      /mnt/disk2     xfs     defaults,noatime  0       2
+     LABEL=DISK3      /mnt/disk3     xfs     defaults,noatime  0       2
+     LABEL=DISK4      /mnt/disk4     xfs     defaults,noatime  0       2
+
+.. note:: 
+
+   For certain cloud environments (Amazon i3, for example), MinIO recommends also setting the ``nofail`` option in ``/etc/fstab``.
+   Cloud instances that do not configure this option may become inaccessible.
+   Setting ``nofail`` in these situations allows continued access to the instance to allow you to rectify the mount points.
```

Comment:
I think @harshavardhana 's concern here is that we are recommending something that may not be necessary in all or most configs.

I would suggest softening the language

```suggestion

.. note::

   Cloud environment instances which depend on mounted external storage may encounter boot failure if one or more of the remote file mounts return errors or failure.
   For example, an AWS ECS instances with mounted persistent EBS volumes may fail to boot with the standards ``/etc/fstab`` configuration if one or more EBS volumes fail to mount.

   You can set the ``nofail`` option to silence error reporting at boot and allow the instance to boot with one or more mount issues.

   You should not use this option on systems which have locally attached disks, as silencing drive errors prevents both MinIO an the OS from responding to those errors in a normal fashion.
```

@harshavardhana does that help limit the applicability here?

Reply:
Yes @ravindk89

Reply:
@harshavardhana I updated the language in the latest commit. Can you approve it?